modules/virt-runbook-cdistorageprofilesincomplete.adoc

```diff
@@ -45,7 +45,7 @@ example:
 +
 [source,terminal]
 ----
-$ oc patch storageprofile local --type=merge -p '{"spec": \
+$ oc patch storageprofile <storage_class> --type=merge -p '{"spec": \
```

Comment:
What's the story behind this change? IIUC, this should be modeled after the upstream runbook: https://kubevirt.io/monitoring/runbooks/CDIStorageProfilesIncomplete.html

Reply:
Can we get QE verification to be safe? I'm sure you are right and I don't recall why I made the change.

Reply:
+1 to QE review; @jpeimer - are you the right QE contact to check this? or @sradco ?

Reply:
@ousleyp I think your change makes sense but I dont see it in the PR anymore.

Reply:
@sradco I put back the change. Thank you.

Reply:
@jpeimer would be the right person to review from QE side, as this is related to storage component.

Reply:
Change looks good, but a small issue I noticed: getting an error when trying to run the patch

```
$ oc patch storageprofile standard-csi --type=merge -p '{"spec": \
  {"claimPropertySets": [{"accessModes": ["ReadWriteOnce"], \
  "volumeMode": "Filesystem"}]}}'
Error from server (BadRequest): error decoding patch: invalid character '\\' looking for beginning of value

```

Without the \ it works fine 
```
$ oc patch storageprofile standard-csi --type=merge --patch='{"spec": {"claimPropertySets": [{"accessModes": ["ReadWriteOnce"], "volumeMode": "Filesystem"}]}}'
storageprofile.cdi.kubevirt.io/standard-csi patched
```